tasks.py.jinja

```diff
@@ -277,13 +277,13 @@ def lint(c, verbose=False):
 
 
 @task(develop)
-def start(c, detach=True, ptvsd=False):
+def start(c, detach=True, debugpy=False):
     """Start environment."""
     cmd = "docker-compose up"
     if detach:
         cmd += " --detach"
     with c.cd(str(PROJECT_ROOT)):
-        c.run(cmd, env=dict(UID_ENV, DOODBA_PTVSD_ENABLE=str(int(ptvsd))))
+        c.run(cmd, env=dict(UID_ENV, DOODBA_DEBUGPY_ENABLE=str(int(debugpy))))
```

Comment:
Do you think we could use this opportunity to add also support for `DEBUGPY_ARGS`?

I see an opportunity for that. It'd be nice if Odoo was started within debugpy always when running in vscode, and then starting the debug session would be more instant.

Reply:
Yes, we could do that. I have seen a comment on the code, however, stating it would also make Odoo slower... Can that make a difference?

Reply:
Yes, it's always slower, but having the option can help.

Depending on what you're doing, you need more speed or you need better debugability.

In any case, it can be done later if you prefer.

Reply:
Well, in that case I was thinking maybe for another time. It is not critical and it would speed up adding debugpy, which is more important.